bin/start_backend.sh

```diff
@@ -147,7 +129,9 @@ if [[ "${RUN_NUMA}" -ne "-1" ]]; then
     set +e
 fi
 
-export LIBHDFS_OPTS=$final_java_opt
+# Appending the option to avoid "process heaper" stack overflow exceptions.
+export JAVA_OPTS_FOR_JDK_9_AND_LATER="$JAVA_OPTS_FOR_JDK_9_AND_LATER -Djdk.lang.processReaperUseDefaultStackSize=true"
```

Comment:
a little bit chaos here about the JAVA_OPTS, LIBHDFS_OPTS and JAVA_OPTS_FOR_JDK_9_AND_LATER

need to make clear about which variables are supported and what's the precedence of these variables.